### ISSUE TITLE
User CSV import success message is wrong

### ISSUE THREAD

**Issue by normanjaeckel** [OpenSlides/OpenSlides#2218]
* Setup new database
* Download CSV example file for users.
* Import this file

You see that 3 new users are to be imported but the success message says that only one or two are imported successfully.

**Comment by matakuka** [comment]
Random observation from such an import:
I imported 7 entries, got three "500 internal server error" on client side. The server side reported
```
Traceback (most recent call last):
  File "/home/.../.virtualenv/lib/python3.4/site-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/.../.virtualenv/lib/python3.4/site-packages/channels/handler.py", line 226, in process_exception_by_middleware
[...]
  File "/home/.../.virtualenv/lib/python3.4/site-packages/django/db/backends/sqlite3/base.py", line 323, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: database is locked
```
All seven entries imported (correctly, as far as I can see), but the message said that four were imported. Thus, the number counts the number of correct reponses from the server, not the number of entries imported into the database. Also, the server sends an error on some occasions that may need attention.

**Comment by normanjaeckel** [comment]
Looks like the same DB error as in #2198.

**Comment by emanuelschuetze** [comment]
Yes, #2198 is an other bug and is not part of this csv import issue.

**Comment by emanuelschuetze** [comment]
This issue works with #2271 and a the fix in current master of channels (fixed by #2198).
